docs/DeveloperGuide.md

```diff
@@ -580,6 +633,32 @@ testers are expected to do more *exploratory* testing.
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is not a valid matriculation number) <br>
       Expected: Similar to previous.
       
+### Editing an appointment 
+
+1. Editing an appointment in the Vax@NUS records. 
+    1. Prerequisites: Sample data of students and appointments are loaded in Vax@NUS.
+    1. Test case: `editAppt A0182345T d/2021-11-13 ts/14:00
+       `
+       Expected: Alex Yeoh's appointment is changed to the given date and time
+       
+    1. Test case: `editAppt A1234567X d/2021-11-13 ts/14:00
+       `
+       Expected: No appointment is edited. Error details shown in the status message telling user that the requested appointment does not exist. 
+       
+    1. Test case: `editAppt A0182345T d/2021-11-130 ts/15:00
+       `
+       Expected: No appointment is edited. Error details shown in the status message telling the user that the date should be of the format `YYYY-MM-DD`.
+       
+    1. Test case: `editAppt A0182345T d/2021-11-13 ts/125:00
+       `
```

Comment:
I think there is a typo here "125:00"

Reply:
That is not a typo. That is done on purpose to show what happens when the wrong time format is entered

Reply:
Oh sorry for the oversight! I'll merge the PR now.